CMake/HPHPFunctions.cmake

```diff
@@ -1,3 +1,5 @@
+cmake_policy(SET CMP0026 OLD)
```

Comment:
Without it newer versions of cmake complains about the use of `get_target_property(... LOCATION)`.  I couldn't figure out an easy way to avoid using it for now so this policy setting tells cmake to use the old behavior.
https://cmake.org/cmake/help/v3.0/policy/CMP0026.html